account-allow-reconciliation/__manifest__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+##############################################################################
+#
+# Allow Reconciliation
+# Copyright (C) 2019 OpusVL (<http://opusvl.com/>)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Allow Reconciliation',
+    'version': '0.1',
```

Comment:
```suggestion
    'version': '11.0.1.0.0',
```

Reply:
This would be displayed as 11.0.0.1 in the module version.